.bingo/ginkgo.mod

```diff
@@ -0,0 +1,5 @@
+module _ // Auto generated by https://github.com/bwplotka/bingo. DO NOT EDIT
+
+go 1.23.4
```

Comment:
Not all the related repos are able to move to 1.23 at this time.
Any reason you cannot use what was in [original repo](https://github.com/operator-framework/catalogd/blob/main/.bingo/ginkgo.mod#L3-L5)?

Reply:
The file says `DO NOT EDIT`
It was created with bingo get. 


We have other scenarios; if we decide to downgrade, we downgrade it to another PR  (we have not decided yet; see: https://github.com/operator-framework/operator-controller/pull/1634). 

So, I think we should keep it as a BINGO added unless we need to change it.

Reply:
I agree with Jordan. The version dumped here depends on the version of golang you are running. Set your base `go` executable to 1.22.x or something similar?

We've been running amok with golang versions.

Reply:
Thanks for addressing!